requirements.txt

```diff
@@ -0,0 +1,8 @@
+calendar
+re
+sys
+abc
+datetime
+msvcrt
```

Comment:
Can we import this in mac ?
Is it working ? (As in ... is it possible to log entries and re-read them afterwards ?)